stoplight.js

```diff
@@ -0,0 +1,10 @@
+console.log("Hello World")
+
+var red = $("#red");
+var yellow = $("#yellow");
+var green = $("#green");
+
+$("#red").click(function() {
+
+    $("#red").css("background", "green");}
```

Comment:
Almost there with red, although you're using green here.

I would encourage you to think about how you could use the provided `active` class to make this work.